content/en/docs/concepts/containers/images.md

```diff
@@ -29,8 +29,7 @@ and possibly a port number as well; for example: `fictional.registry.example:104
 
 If you don't specify a registry hostname, Kubernetes assumes that you mean the Docker public registry.
 
-After the image name part you can add a _tag_ (as also using with commands such
-as `docker` and `podman`).
+After the image name part you can add a _tag_ (in the same way you would when using with commands like `docker` or `podman`).
```

Comment:
nit: Tidy up the sentence?
A container image name may include a `tag` component. To add a `tag` to an image, you can use `docker` or `podman`.